src/rs/src/text.rs

```diff
@@ -371,9 +371,9 @@ impl<'a> SkillThing<'a> {
 			Skill::freeevade =>
 				Cow::from("If your opponent has a shield, your airborne creatures have a 25% chance to bypass the shield. Otherwise, your creatures have a 25% chance to deal 50% more damage. Your creatures have 20% chance to evade opponent's targeted spells & skills"),
 			Skill::freeze(x) =>
-				Cow::from(format!("Freeze target creature or weapon for {} turns. Frozen cards cannot attack or use active skills, & do not activate per-turn skills", x)),
+				Cow::from(format!("Freeze target creature or weapon for {} turns. Frozen cards cannot attack or use active skills & do not activate per-turn skills", x)),
```

Comment:
[The oxford comma should always be used to minimize ambiguity](https://en.wikipedia.org/wiki/Serial_comma)

Reply:
Not an Oxford comma (requires trio+). Understood, and will modify into proper series statement; respecting ampersand.